crates/pallet-domains/src/tests.rs

```diff
@@ -261,6 +240,24 @@ pub(crate) fn new_test_ext() -> sp_io::TestExternalities {
     t.into()
 }
 
+pub(crate) struct MockDomainFraudProofExtension(Randomness, Moment);
```

Comment:
For more than one field I generally prefer named fields for readability, tuple is fine for new types though.